ssm_jax/hmm/models/autoregressive_hmm.py

```diff
@@ -0,0 +1,96 @@
+import jax.numpy as jnp
+import jax.random as jr
+
+from ssm_jax.hmm.models.linreg_hmm import LinearRegressionHMM
+
+from jax import vmap, lax, tree_map
+
+class LinearAutoregressiveHMM(LinearRegressionHMM):
+    """A linear autoregressive HMM (ARHMM) is a special case of a
+    linear regression HMM where the covariates (i.e. features)
+    are functions of the past emissions.
```

Comment:
Please explain the form of the model in math. 
Is it
```
 p(z(1:T), y(1:T) ) = prod_t p(z(t) | z(t-1)) p(y(t) | z(t), y(t-L:t-1))
where
 p(y(t) | z(t)=k, y(t-1)) = sum_{lag=1}^L N(y(t)| W_{k,lag} y(t-lag) + b_{k,lag}, Sigma_{k,lag})

```

Reply:
That’s right. I’ll add it to the doc string.